examples/motoko_examples/calc/src/calc.did

```diff
@@ -4,4 +4,4 @@ service: {
     "mul": (int) -> (int);
     "div": (int) -> (opt int);
     "clearall": () -> ();
-}
+}
```

Comment:
We should have newlines at the end of our files, @dansteren can help you set up VS Code automation for this if you want

Reply:
Also why is this file even in the pull request?